UnitTests/Tests/FISUserSpec.m

```diff
@@ -0,0 +1,60 @@
+//
+//  FISUserSpec.m
+//  facepad
+//
+//  Created by Tom OMalley on 7/21/15.
+//  Copyright 2015 The Flatiron School. All rights reserved.
+//
+
+#import "Specta.h"
+#import "FISUser.h"
+#import "FISImagePost.h"
+#import "FISTextPost.h"
+
+
+SpecBegin(FISUser)
+
+describe(@"FISUser", ^{
+    
+    __block FISUser *testUser;
+    __block FISImagePost *imagePost1;
+    __block FISTextPost *textPost1;
+    
+    beforeAll(^{
+        testUser = [[FISUser alloc] init];
+        testUser.username = @"jmburgess";
+        testUser.profilePic = [UIImage imageNamed:@"joe-burgess"];
+        testUser.cover = [UIImage imageNamed:@"coverimage"];
+        
+        textPost1 = [[FISTextPost alloc] init];
+        textPost1.textContent = @"Excited for the first day of school!";
+        textPost1.user = testUser;
+        
+        imagePost1 = [[FISImagePost alloc] init];
+        imagePost1.imageContent = [UIImage imageNamed:@"flying"];
+        imagePost1.user = testUser;
+        
+        testUser.posts = [@[textPost1, imagePost1] mutableCopy];
+    });
+    
+    it(@"has properties for username, profile picture, cover image, and posts, with no extra stuff modifying their setters",
```

Comment:
If you have an and in your `it` usually it's testing too much. This is testing *way* too much. You usually just want one, maybe two or three `expect`s per `it` block.